Date: 2005-05-31
Initial Package Version: 2.9.5
Upstream Status: http://bugs.archlinux.org/index.php?do=details&id=2785
Origin: Christian Hamar alias krix <krics@linuxforum.hu>
Description: Without the attached patch pacman counts wrong md5sum on x86_64.
diff -Naur pacman-2.9.5.orig/src/md5.h pacman-2.9.5/src/md5.h
--- pacman-2.9.5.orig/src/md5.h	2005-05-31 14:24:01.000000000 +0000
+++ pacman-2.9.5/src/md5.h	2005-05-31 14:25:14.000000000 +0000
@@ -31,7 +31,7 @@
 typedef unsigned short int UINT2;
 
 /* UINT4 defines a four byte word */
-typedef unsigned long int UINT4;
+typedef unsigned int UINT4;
 
 
 /* MD5 context. */
